tfhe-zk-pok/src/proofs/mod.rs

```diff
@@ -127,6 +128,8 @@ impl<G: Curve> GroupElements<G> {
     }
 }
 
+pub const HASH_METADATA_LEN_BYTES: usize = 256;
```

Comment:
is that required for the performance ?

If not then I would have that in a separate PR please

Reply:
will do